src/clj_kondo/impl/core.clj

```diff
@@ -61,10 +61,9 @@
 (defn opts [^java.io.File cfg-file]
   (let [include #(read-fn cfg-file read-edn-file %)]
     {:readers
-     {'include include
+     {'include include}}))
```

Comment:
Can you revert this? I'd like to keep those comments within the map 🤣

Reply:
Sorry, parinfer :)